queries/just/textobjects.scm

```diff
@@ -9,6 +9,9 @@
 (dependency_expression
   (_) @parameter.inside) @parameter.outer
 
+(function_call
+  arguments: (sequence (expression) @parameter.inside) @parameter.outer) @function.outer
+
```

Comment:
Was this change intended? Any sort of query changes should go in `queries-src`, this file gets autogenerated

Reply:
Nope, I must have merged poorly.

Reply:
Update, this call seems to be produced automatically by running `just gen` on the current `HEAD` of this PR